core/templates/dev/head/pages/base.html

```diff
@@ -138,10 +138,9 @@
         <div class="oppia-content-container">
```

Comment:
I can't seem to leave a file-level comment, but in audio_expand.png the text should not be done as an image -- rather, use CSS styling to create the tab. Otherwise it's not internationalizable.

Reply:
done